hivelocity/provider.go

```diff
@@ -22,14 +22,15 @@ func Provider() *schema.Provider {
 				Type:        schema.TypeString,
 				Required:    true,
 				Sensitive:   false,
-				Description: descriptions["api_key"],
+				Description: descriptions["api_url"],
 				DefaultFunc: schema.EnvDefaultFunc("HIVELOCITY_API_URL", "https://core.hivelocity.net/api/v2"),
 			},
 		},
 		DataSourcesMap: map[string]*schema.Resource{
 			"hivelocity_bare_metal_device": dataSourceBareMetalDevice(),
 			"hivelocity_device":            dataSourceDevice(),
 			"hivelocity_product":           dataSourceProduct(),
+			"hivelocity_initial_creds":     dataSourceInitialCreds(),
```

Comment:
Based on the suggestion naming conventions in the Terraform docs for creating a provider.  We want to name this hivelocity_device_initial_creds

Reply:
Done.